server/base/src/main/java/org/apache/accumulo/server/conf/TableConfWatcher.java

```diff
@@ -33,11 +33,13 @@
   private static final Logger log = Logger.getLogger(TableConfWatcher.class);
   private final Instance instance;
   private final String tablesPrefix;
+  private final int tablesPrefixLength;
   private ServerConfigurationFactory scf;
 
   TableConfWatcher(Instance instance) {
     this.instance = instance;
     tablesPrefix = ZooUtil.getRoot(instance) + Constants.ZTABLES + "/";
+    tablesPrefixLength = tablesPrefix.length();
```

Comment:
I think this got carried in from my initial patch that I had attached to the JIRA that @milleruntime picked up. I think it was part of an unrelated change that got picked up in the diff. I actually discarded that change from my workspace because I couldn't remember why I had originally done this. :smile: